src/components/Trade/hooks/useSwapper/useSwapperV2.ts

```diff
@@ -144,6 +161,7 @@ export const useSwapper = () => {
       return bestTradeSwapper.buildTrade({
```

Comment:
<!-- GRAPHITE_METADATA {"trueLocation":{"endLine":154,"prLink":"https://app.graphite.dev/github/pr/shapeshift/web/2842"}} -->
<i class='graphite__hidden'>[Re: line 154]<br/></i>

This should be programmatic too<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/shapeshift/web/2842?utm_source=unchanged-line-comment">Graphite</a>.</span>